src/components/ScheduleRequest/index.js

```diff
@@ -29,6 +29,34 @@ function ScheduleRequests({ match }) {
     Authorization: `Bearer ${token}`
   };
 
+  const requestApproval = e => {
+   
+    e.preventDefault();
+    const token = window.localStorage.getItem('token');
+    const requestId = e.target.id;
+    const action = e.target.classList[1] === 'Approve' ? 'Approved' : 'Rejected';
+
+    const headers = {
+      'Content-Type': 'application/json',
+      Authorization: `Bearer ${token}`
+    };
+
+    axios({
+      method: 'PUT',
+      url: `http://localhost:6060/api/v1/request/${requestId}?status=${action}`,
+      headers
+    })
```

Comment:
This axios method has been abstracted into `sendPutRequest` method in the action folder that handles this

Reply:
noted!!!!!